added argument to success function which fixed functionality

Gabe-DiCarlo opened the pull request:
I added an argument to your success function in your index.js file. The app wasn't displaying any data before that, but once I did that it fixed it. 

Another problem that I ran into with getting the app to run properly on my machine was that it kept throwing errors and saying I was denying the user location data permissions. After researching it I found that this is caused by the localhost being considered insecure, I found this link which fixed it in case any of you have the same problem. 
https://stackoverflow.com/questions/49896961/a-geolocation-request-can-only-be-fulfilled-in-a-secure-context

With what you are trying to do with your app I found a really useful site which is where I found the solution to passing that argument in your success function, it may be useful as you continue to develop it.
https://www.w3spoint.com/html5-geolocation

    Analysis of the program.
This program is a weather app which will show users the weather in their location and allow them to look up current weather and weather forecasts in places they search. It is currently at the stage where it will retrieve users location data.

    Was the program available in UC Github on time?
Yes
    Is the program documented/commented well enough for you to understand?
Yes
    Does the program compile?
Yes

Technical Concepts:
1. I learned how to use HTML5 geolocation functions
2. I learned how to bypass security features in google chrome when accessing localhost during development
3. I gained more experience in using Chrome developer tools to troubleshoot the exact problem in the program when the eclipse debugger wasn't helping much.

provanpm commented:
You say you added a parameter to the success function, but what you actually did was rename it from ‘pos’ to ‘position’ (line 15 and 17, then 17 and 19 in the files changed section). Was the parameter name producing an error? Not able to recreate this issue.

Gabe-DiCarlo commented:
No I tried it again and you're right. I thought that the position parameter was using a built in method for the HTML5 geolocation service, kind of like the success and error parameters. I think that while troubleshooting I changed that while reading up on it at the link I provided and I changed the browser settings to allow location access and I didn't realize that I just needed to change the browser settings. Sorry about the confusion.
